AdobeIms/view/adminhtml/web/js/signIn.js

```diff
@@ -14,14 +14,18 @@ define([
         defaults: {
             profileUrl: 'adobe_ims/user/profile',
             logoutUrl: 'adobe_ims/user/logout',
+            dataProvider: 'name = adobe_stock_images_listing.adobe_stock_images_listing_data_source, ns = adobe_stock_images_listing',
```

Comment:
AdobeIms signIn implementation cannot rely on listing data provider or any other part of AdobeStock implementation. Please apply the changes to AdobeStockImageAdminUi module